pkgs/development/libraries/cctz/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchFromGitHub, darwin }:
+{ lib, stdenv, fetchFromGitHub, Foundation }:
```

Comment:
May I suggest to revert the input param from "Foundation" to "darwin"? This solves the mentioned problem...

Reply:
@SuperSandro2000

Reply:
Hi @maj0e. I have not forgotten this here. I asked again for Sandro to have a look. In case he does not answer (he usually does), we can revert it.

Reply:
```
libphonenumber = callPackage ../development/libraries/libphonenumber {
    inherit (darwin.apple_sdk.frameworks) Foundation;
};
```

Reply:
Hi, thanks for the feedback. I finally found some time to fix this, like you suggested. I added the inherit to the top-level and updated the PR. The build works fine now for pkgs and pkgsStatic.